bindings/rust/Cargo.toml

```diff
@@ -27,6 +42,9 @@ serde_yaml = "0.9.17"
 bindgen = { git = "https://github.com/rust-lang/rust-bindgen" , rev = "0de11f0a521611ac8738b7b01d19dddaf3899e66" }
 cc = "1.0"
 
+[target.'cfg(target_env = "msvc")'.build-dependencies]
+glob = "0.3"
+
```

Comment:
Just curious, what's different with MSVC here?

Reply:
it seems to be blst on windows is compiled with it's [own assembly files](https://github.com/supranational/blst/tree/master/build/win64). This is taken from [`blst rust bindings`](https://github.com/supranational/blst/blob/e9dfc5ee724b5b25d50a3b6226bee8c2c9d5e65d/bindings/rust/build.rs#L8-L26)

Reply:
Oh that makes sense. It needs glob to find the files. Thanks.